ProxyAppsSuite/tests/miniamr.yaml

```diff
@@ -0,0 +1,895 @@
+_base:
+   summary: 'Apply a stencil calculation on a unit cube computational domain'
+   build:
+      summary: building the miniAMR test
```

Comment:
remove summary key from build section, it breaks the test, won't run unless removed.

Reply:
in its place, add `timeout: 300` as it times out after 30 seconds- something I hit when running git clone